libs/gi/page-team/src/TeamSetting/index.tsx

```diff
@@ -191,6 +249,14 @@ function TeamEditor({
           character" buffs, and cannot be empty.
         </Trans>
       </Alert>
+      <Grid container sx={{ justifyContent: 'space-between' }} spacing={2}>
+        <Grid item>
+          <Typography variant="h5">Selected Team Members</Typography>
+        </Grid>
+        <Grid item>
+          <Button onClick={() => setShowMultiSelect(true)}>Quick Select</Button>
```

Comment:
Maybe add a icon to the button, I would recommend https://mui.com/material-ui/material-icons/?query=multiple&selected=Filter4

Reply:
done